google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -146,9 +146,9 @@ public String getAppProfileId() {
     return mutateRowSettings;
   }
 
-  /** Returns the object with the settings used for calls to MutateRows. */
-  public BatchingCallSettings<RowMutation, Void> mutateRowsSettings() {
-    return mutateRowsSettings;
+  /** Returns the object with the settings used for manually batched calls to MutateRows. */
+  public BatchingCallSettings<RowMutation, Void> bulkMutateRowsSettings() {
```

Comment:
The `ResponseT` of the call settings doesn't match the `ResponseT` of the callable in the stub.

Reply:
The 2 callables that depend on these settings are
```
UnaryCallable<BulkMutation, Void> bulkMutateRowsCallable
UnaryCallable<RowMutation, Void> bulkMutateRowsBatchingCallable
```
Both use `Void` for `ResponseT`. However, the first one differs in `RequestT`. This is the result of them sharing the same underlying `ServerStreamingCallable<MutateRowsRequest, MutateRowsResponse> mutateRowsCallable()` from GAPIC, so their retry codes (that are passed to the underlying exception callable) must be identical. So keep them consistent, I chose to use a single settings object that is the super set of the 2 user-facing callables.

An alternative would be for me to create my own BatchingSettings class that only contains the flow & batching settings. But I think that would be more confusing

Reply:
Maybe add a comment about the mismatch.